toolkit/imageconfigs/variants/baremetal.yaml

```diff
@@ -0,0 +1,15 @@
+os:
+  hostname: azure-linux
+  selinux:
+    mode: enforcing
+  kernelCommandLine:
+    extraCommandLine: "console=tty0 console=ttyS0 rd.info log_buf_len=1M"
+  packages:
+    installLists:
+    - packagelists/baremetal-packages.yaml
+    - packagelists/base-image-packages.yaml
+    - packagelists/cloud-init-packages.yaml
+    - packagelists/selinux.yaml
+scripts:
+  finalizeCustomization:
+  - path: ../scripts/cleanup.sh
```

Comment:
Given that we would want to package the variants definitions together, it may be a good idea to create a copy of those scripts in this directory.